src/index.js

```diff
@@ -146,9 +158,10 @@ const Clap = class extends React.Component {
     })
   }
 
-  render() {
+  render () {
     const { count, countTotal, isClicked, isHover } = this.state
-    const { iconComponent: ClapIcon } = this.props
+    const { iconComponent: ClapIcon, canSwitch } = this.props
+    const clapCountText = canSwitch ? `${isClicked ? '+' : '-'}1` : `+${count}`
```

Comment:
please change the hard-coded count `+1`.

What if there is the case where some synchronization is used to make a real-time update for the `clapCountText`?

If that so we have to compare the current count with previous count, cannot just depend on `isClicked`